Framework/Geometry/inc/MantidGeometry/Instrument/ParameterMap.h

```diff
@@ -386,9 +381,6 @@ class MANTID_GEOMETRY_DLL ParameterMap {
   /// Pointer to the DetectorInfo object. NULL unless the instrument is
   /// associated with an ExperimentInfo object.
   boost::shared_ptr<const Beamline::DetectorInfo> m_detectorInfo{nullptr};
-  /// Pointer to the ComponentInfo object. NULL unless the instrument is
-  /// associated with an ExperimentInfo object.
-  boost::shared_ptr<const Beamline::ComponentInfo> m_componentInfo{nullptr};
```

Comment:
With this gone, how are you going to implement `Component::getPos()` to forward to `ComponentInfo`?